extra/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+set -e
+
+files_ownership () {
+    chown -hRc "${PUID=1000}":"${PGID=1000}" /app/data
```

Comment:
Consider renaming the variables to 

from PUID to UID
from PGID to GID

Reply:
You shouldn't: Try `echo $UID` at your PC. The UID variable is currently used in linux systems, so host will use it too. The common practice (all [linuxserver.io](https://linuxserver.io) images, [Mattermost](https://docs.mattermost.com/install/prod-docker.html) etc.) is naming them `PUID` and `PGID` for so-called _easy user mapping_, so I decided to stick to this convention.